addons/hr_holidays/models/hr_leave_type.py

```diff
@@ -93,8 +93,8 @@ def _model_sorting_key(self, leave_type):
         ('officer', 'Approved by Time Off Officer'),
         ('set', "Set by Time Off Officer")], default='officer', string='Approval')
     has_valid_allocation = fields.Boolean(compute='_compute_valid', search='_search_valid', help='This indicates if it is still possible to use this type of leave')
-    time_type = fields.Selection([('leave', 'Time Off'), ('other', 'Other')], default='leave', string="Kind of Leave",
-                                 help="Whether this should be computed as a holiday or as work time (eg: formation)")
+    time_type = fields.Selection([('other', 'Worked Time'), ('leave', 'Absence')], default='other', string="Kind of Time Off",
+                                 help="The distinction between working time (ex. Attendance) and absence (ex. Formation) will be used in the computation of Accrual's plan rate.")
```

Comment:
The help message is missing from the .po file as well

Reply:
Done, thanks :-)